typings.d.ts

```diff
@@ -465,25 +468,27 @@ declare module 'contentful-ui-extensions-sdk' {
     ids: Pick<IdsAPI, 'environment' | 'space' | 'extension' | 'user'>
   }
 
+  interface AppConfigAPI {
+    /** Tells the web app that the app is loaded */
+    setReady: () => Promise<void>
+    /** Returns true if an App is installed **/
+    isInstalled: () => Promise<boolean>
+    /** Returns parameters of an App, null otherwise **/
+    getParameters: () => Promise<null | Object>
+    /** Returns current state of an App, null otherwise **/
+    getCurrentState: () => Promise<null | Object>
```

Comment:
I think it would be nice to allow the user to define the type instead of having to use `as`.
```suggestion
    getCurrentState<T>: () => Promise<null | T>
```

This comment applies to this and `getParameters` above.